backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc

```diff
@@ -0,0 +1,77 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oadp-kubevirt"]
+// If docs team approves the name, "Configuring OADP with Kubevirt" great but I suspect it will need to be "Configuring OADP with OpenShift Virtualization" or "Configuring OADP with Virtualization" or perhaps... bear with me.. "OADP and OpenShift-Virtualization"
+= Configuring the OpenShift API for Data Protection with {VirtProductName}
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-kubevirt
+:installing-oadp-kubevirt:
+:credentials: cloud-credentials
+:provider: kubevirt
+
+toc::[]
+
+You install the {oadp-first} with {VirtProductName} by installing the OADP Operator and configuring a backup location and a snapshot location. Then, you install the Data Protection Application.
```

Comment:
I think we can remove any reference to VSL since we only support CSI + CSI w/ DM